app/Models/Beatmapset.php

```diff
@@ -1275,11 +1275,16 @@ private function getBBCode()
         return new BBCodeFromDB($description, $post->bbcode_uid, $options);
     }
 
+    public function getArtistUnicodeAttribute()
+    {
+        return $this->attributes['artist_unicode'] ?? $this->artist;
```

Comment:
Also for the other one

```suggestion
    public function getArtistUnicodeAttribute($value)
    {
        return $value ?? $this->artist;
```